### PR TITLE
RU locale update for v0.7.3

### PR DESCRIPTION
**Pull request by Astorin**

### CODE REVIEW

File: locale/ru/loader_modernized.cfg [hunk 1]
```diff
@@ -1,81 +1,83 @@
...
+mdrn-enable-stacking=Включить складывание в стопки
+mdrn-use-aai-graphics=Использовать графику AAI Loaders
+
+[mod-setting-description]
+mdrn-use-electricity=Если включено, погрузчики требуют электропитание для работы.
+mdrn-double-recipe=Удваивает стоимость рецепта и производит 2 погрузчика одновременно, чтобы подземные конвейеры расходовались парами.
+mdrn-migrate-from-miniloaders=Попытка перенести построенные мини-погрузчики в Loaders Modernized. Погрузчики в инвентаре и контейнерах будут потеряны. Рецепты сборщиков будут сброшены.
+mdrn-enable-chute=Включить рецепт, чтобы создавать очень медленный скат в стиле погрузчика.
+mdrn-enable-stacking=Включить складывание предметов в стопки на конвейере, согласно уровню технологии складывания.
+mdrn-use-aai-grphics=Обнаружен мод AAI Loaders.\nИспользуется только графика из AAI Loaders для Loaders Modernized.\n\nЕсли вы установите значение "false" и перезапустите, вы сможете использовать и Loaders Modernized, и AAI Loaders.
```
Comment (kryojenik) [inline locale/ru/loader_modernized.cfg:69]: I had a typo in my original file - sorry.  Should be 'mdrn-use-aai-graphics'

**Review by kryojenik — CHANGES_REQUESTED** [review]
Thanks for the Russian translations!

Could you please return the line endings to LF and not change them to CRLF

I also added two additional strings in the mean time.
mdrn-cheap-stacking=Cheaper stacking loaders
mdrn-cheap-stacking=When belt stacking is enabled, use the cheaper non-stacking recipes for loaders.

File: locale/ru/loader_modernized.cfg (hunk 1 above)
Comment (Astorin) [reply locale/ru/loader_modernized.cfg:69]: Hello.
I don't see any changes to the English locale to change the translation accordingly.
What's the problem with the CRLF? It's always been fine in all translations.

**Review by Astorin — COMMENTED** [review]
All seems ok

File: locale/ru/loader_modernized.cfg (hunk 1 above)
Comment (kryojenik) [reply locale/ru/loader_modernized.cfg:69]: Nothing wrong with the CRLF from a functional, technical point of view.  I've built the original files as UNIX/Mac line ending and switching them to DOS base line endings causes every line to be marked as changed in the merge.  It's a style thing and wanting to keep the commits not pulling in line-ending changes.

**Comment by kryojenik** [comment]
And for the two new strings, I hadn't pushed my local commits yet.  The should be there now.

**Comment by kryojenik** [comment]
image

**Comment by Astorin** [comment]
I edit files directly on GitHub, I don't see where I can change CRLF to LF.

**Review by kryojenik — APPROVED** [review]

**Pull request merged**